packages/phone-number-privacy/combiner/src/pnp/endpoints/sign/action.ts

```diff
@@ -134,6 +143,12 @@ export function pnpSign(
       }
     }
     const error = errorCodeToError(errorCode)
+    if (error === ErrorMessage.NOT_ENOUGH_PARTIAL_SIGNATURES) {
+      Counters.errors.labels(request.url, error).inc()
+      Counters.notEnoughSigErrors.labels(request.url).inc()
+    } else if (error in WarningMessage) {
+      Counters.warnings.labels(request.url, error).inc()
+    }
```

Comment:
Should we have an additional `counters.errors` for anything other than partial sig errors?